minetest/minetest
# WIP: Add gameid aliases

Allows defining gameid aliases, so that a game can be completely renamed.

Fixes #13662 

Roadmap: 2.2 (refactoring), sort of.

Note: This does not work yet.

## To do

This PR is a Work in Progress

## How to test

1. Create a world under a game.
2. Rename the folder of the game.
3. Define the old folder name as `gameid_alias` in the game.conf file.
4. Restart Minetest.
5. Verify if you can still see the world under the renamed game.
